singer-io/tap-outreach
# Change pagination from offset to cursor, rate limiting.

# Description of change
Change from `offset` to `cursor` [pagination](https://api.outreach.io/api/v2/docs#pagination) in `sync.py`. Add [rate limit](https://api.outreach.io/api/v2/docs#rate-limiting) decorator to `client.py`.

# Manual QA steps
Ran singer-discover, singer-check-tap, and target-stitch (initial load and incremental).
 
# Risks
Low - removes potential 10,000 limit and possible infinite loop w/ offset pagination. Tested multiple pages but not with large data volumes.
 
# Rollback steps
Revert to 0.3.0
